renderers/src/main/java/com/pedrogomez/renderers/RVRendererAdapter.java

```diff
@@ -56,7 +61,15 @@ public T getItem(int position) {
     return position;
   }
 
-  /**
+  public void setCollection(AdapteeCollection<T> collection) {
+      if (collection == null) {
+          throw new IllegalArgumentException("Collection must be not null");
+      }
+
+      this.collection = collection;
+  }
+
+    /**
```

Comment:
As some of the library users are still using ``ListView`` we have to update the ``RendererAdapter`` class with the same behaviour and tests :)